util/include/crypto/openssl/digest_creator.hpp

```diff
@@ -73,5 +74,5 @@ class OpenSSLDigestCreator : public concord::crypto::DigestCreator {
 
 namespace concord::crypto {
 
-using DigestGenerator = concord::crypto::openssl::OpenSSLDigestCreator<concord::crypto::openssl::SHA2_256>;
+using DigestGenerator = concord::crypto::openssl::OpenSSLDigestCreator<concord::crypto::SHA2_256>;
```

Comment:
same here, the implementation of openssl should be explicit with the types it uses (not concord::crypto but rather concord::crypto::openssl)

Reply:
`concord::crypto` is used instead of `concord::crypto::openssl` as a wrapper. This is more cleaner way of writing. Wdyt?

Reply:
Same argument as before. This code is **internal** to the specific openssl implementation of the interface, thus it can't use the alias which might change.

Reply:
Addressed in https://github.com/vmware/concord-bft/pull/2869/commits/4a231052ce407519d8d8e3b592965876607f3a23